### PR TITLE
[Maintenance] Updating stub and params with new name style

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @abhi18av following my maintenance tasks, I'm updating the stub profile with a standard container and also updating the params names as we discussed on the `dandrea_legionella` pipeline

## PR Content:
* Update to stub profile
* new params name
* spliting mtbseq into 2 modules

## PR ending
As always, feel free to make questions, discuss or criticize my changes, I'll be happy on hear your thoughts.
Hope we have a merge!

### CODE REVIEW

File: modules/mtbseq/mtbseq.nf [hunk 1]
```diff
@@ -4,13 +4,13 @@ nextflow.enable.dsl = 2
 // - tar -xvf GATK_TAR_FILE
 // - gatk-register gatk_folder/gatk_jar
 
-params.resultsDir_mtbseq_per_sample = "${params.outdir}/mtbseq/samples"
-params.saveMode_mtbseq_per_sample = 'copy'
-params.shouldPublish_mtbseq_per_sample = true
+params.results_dir_mtbseq_per_sample = "${params.outdir}/mtbseq/samples"
+params.save_mode_mtbseq_per_sample = 'copy'
+params.should_publish_mtbseq_per_sample = true
```
Comment (abhi18av) [inline modules/mtbseq/mtbseq.nf]: @Mxrcon , would it be possible to separate the MTBSEQ processes into 2 modules, this we we can continue using the same pattern of `params.results_dir` all across.

Comment (abhi18av) [reply modules/mtbseq/mtbseq.nf]: Once that is updated, we also need to update the config/params accordingly.

**Review by abhi18av — CHANGES_REQUESTED** [review]
Hi @Mxrcon ,

Thanks for taking the time togive these touches, I have requested a minor change for the MTBSEQ modules so that we can have independent processes and therefore can use the generic params for the module.

File: modules/mtbseq/mtbseq.nf (hunk 1 above)
Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: Yes! I agree on that, I'll implement that and update the cofigurations/parametes after and them ping you again here!

**Review by abhi18av — APPROVED** [review]
Looks good to merge ✅ 

Thanks @Mxrcon!

**Comment by Mxrcon** [comment]
Hey @abhi18av This pr wasn't merged also :rofl: :rofl:

**Comment by abhi18av** [comment]
😅 , thanks for the reminder buddy!

**Comment by Mxrcon** [comment]
No problem :laughing:

**Pull request merged**